readme.md

```diff
@@ -118,19 +118,21 @@ You can pass a query parameter `&hide=` to hide any specific stats with comma-se
 ![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&hide=contribs,prs)
 ```
 
-### Adding private contributions count to total commits count
+### Adding private contributions count to the total contributions count
 
-You can add the count of all your private contributions to the total commits count by using the query parameter `&count_private=true`.
-
-> **Note**
-> If you are deploying this project yourself, the private contributions will be counted by default. If you are using the public Vercel instance, you need to choose to [share your private contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile).
+You can include private contributions in the total contributions count by using the query parameter `&count_private=true`. To enable this, you need to choose to [share your private contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile).
 
 > Options: `&count_private=true`
 
 ```md
 ![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&count_private=true)
 ```
 
+> **Warning**
+> Contributions and commits are [not the same](https://github.com/anuraghazra/github-readme-stats/discussions/1679).
+>
+> The best way to get the correct total commits (with private commits included) is to remove the `count_private` property and [deploy on your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance). The private commits will then be counted by default.
+
 ### Showing icons
```

Comment:
Is this needed after the updated logic in https://github.com/anuraghazra/github-readme-stats/commit/c90c1945888a16ea4a098ac240de979fa54a0fa2?